frontend/public/js/script.js

```diff
@@ -45,7 +45,7 @@ $(document).ready(function () {
         position: 'bottom left',
         theme: 'bootstrap'
     });
-
+*/
```

Comment:
Why did you comment out these ? This is why I am not able to access color-pickers.